training/src/entities/answer.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { BaseEntity, Entity, ManyToOne, PrimaryKey, Property } from "@mikro-orm/core";
+import { v4 } from 'uuid';
+import { Offer } from "./offer.entity";
+import { Question } from "./question.entity";
+
+@Entity()
+export class Answer extends BaseEntity<Answer, 'id'> {
+  @PrimaryKey({ columnType: 'uuid' })
+  id: string = v4();
+
+  @Property()
+  public answer: string;
```

Comment:
I would rename it, its a bit confusing to have `answer` field inside the answer entity